### PR TITLE
side menu three dots fix

### PR DESCRIPTION
**Pull request by benbaler**
https://github.com/hibobio/hibob/issues/25586

just fix for simple open close menu, without the solution for pressing sharing and etc.

### CODE REVIEW

File: projects/ui-framework/src/lib/navigation/side-menu/side-menu-option/side-menu-option.component.ts [hunk 1]
```diff
@@ -75,8 +76,8 @@ export class SideMenuOptionComponent implements OnChanges {
     $event.stopPropagation();
   }
 
-  setShowActions(status: boolean, timeout: number = 0): void {
-    setTimeout(() => this.showActions = status, timeout);
+  setShowActions(status: boolean): void {
```
Comment (danazelniker) [inline projects/ui-framework/src/lib/navigation/side-menu/side-menu-option/side-menu-option.component.ts]: do you still use this function? I see you remove the call from the html

Comment (benbaler) [reply projects/ui-framework/src/lib/navigation/side-menu/side-menu-option/side-menu-option.component.ts]: yes i use it in the html in other places

File: projects/ui-framework/src/lib/navigation/side-menu/side-menu-option/side-menu-option.component.ts [hunk 2]
```diff
@@ -75,8 +76,8 @@ export class SideMenuOptionComponent implements OnChanges {
...
+    setTimeout(() => this.showActions = status);
```
Comment (danazelniker) [inline projects/ui-framework/src/lib/navigation/side-menu/side-menu-option/side-menu-option.component.ts]: check if you need timeout

Comment (benbaler) [reply projects/ui-framework/src/lib/navigation/side-menu/side-menu-option/side-menu-option.component.ts]: you right

**Pull request merged**